docs/contributing/templates/saas-cloud-app.md

```diff
@@ -0,0 +1,53 @@
+---
+id: saas-cloud-otel
+title: Saas App (OTEL) Template
+sidebar_label: SaaS App (OTEL) Template
+description: Using the Sumo Logic SaaS App (OTEL) Template.
+---
+
+:::tip
+Copy this file and edit it for your App integration.
+:::
+
+<img src={useBaseUrl('img/integrations/folder-name/app-name.png')} alt="icon" width="75"/>
+
+Introduction
+
+## Prerequisites (If any)
+
+<List downn the pre-requisites>.
+
+## Log types
+
+### Sample log message
+
+### Sample query
+
+## Collecting Logs for \<App Name\> app
+
+This section provides instructions for setting up [Cloud-to-Cloud-Integration for \<App Name\> Source](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/\<App-Name\>-source/) to create the source and use the same source category while installing the app.
+
+## Installing the \<App Name\> app
+
+This section describes how to install the \<App Name\> app for Sumo Logic and description of the dashboard. You'll also need to configure the [\<App Name\> Cloud-to-Cloud Source](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/\<App-Name\>-source/).
+
+{@import ../../reuse/apps/app-install.md}
+
+## Viewing \<App Name\> dashboards
+
+{@import ../../../reuse/filter-dashboards.md}
+
+## \<Dashboard name\>
+
+\<Copy, paste, and fill in this dashboard section for each dashboard. Enter a dashboard description from above.\> Use this dashboard to:
+
+* Enter a list of features. Example below.
+
+Use this dashboard to:
+
+* Monitor high severity threats and scan attacks.
+* Review \<fill in\> for troubleshooting configuration issues.
+* Understand how to fine-tune \<fill in your product name\> based on \<fill in panel names\>.
+
+\<Add your dashboard screenshot.\>
+<img src='https:///Dashboard-S3-link.png' style={{border: '1px solid black'}} alt="Dashboard-name" />
```

Comment:
not sure if `https:///Dashboard-S3-link.png` is a valid link

Reply:
Yes! this is an invalid link.